styles.css

```diff
@@ -0,0 +1,881 @@
+    flex-direction: column;
+    position: absolute;
+    background-color: #131313;
+    margin-top: 20px;
+  }
+
+  .eventdropdown:hover .eventscontent {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    background-color: #131313;
+    margin-top: 20px;
+  }
+
+  .eventscontent a:hover {
+    color: white;
+  }
+
+  .topmenul li a:hover {
+    color: white;
+  }
+
+  .topmenur li a:hover {
+    color: white;
+  }
+
+  .boxes {
+    flex-direction: row;
+    justify-content: space-evenly;
+  }
+
+  .fundbox {
+    max-width: 300px;
+    margin: 3px;
+  }
+
+  #fundbox4 {
+    display: none;
+  }
+
+  .morefund {
+    flex-flow: row;
+  }
+
+  .getupdates {
+    padding-left: 10px;
+    padding-right: 10px;
+    margin: 10px;
+    border-left: 1px solid #f42;
+    border-right: 1px solid #f42;
+    font-family: Avalon, sans-serif;
+    font-size: 15px;
+    color: #f42;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .dropdown {
+    display: none;
+    width: 66px;
+  }
+
+  #fundbox4 {
+    display: flex;
+  }
+
+  .fundtittle {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+}
+
+@font-face {
+  font-family: tnw;
+  src: url("Audiowide-Regular.ttf");
+}
+
+div {
+  margin: 10px auto;
+}
+
+.highlights {
+  position: relative;
+  top: 100px;
+  width: 98%;
+  margin: auto;
+  margin-bottom: 350px;
+}
+
+.feature-logo {
+  display: none;
+  font-family: tnw, 'Times New Roman', Times, serif;
+}
+
+.main-feature,
+.sub-feature {
+  height: 150px;
+}
+
+.main-feature h3,
+.sub-feature h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.menu-links a {
+  text-transform: uppercase;
+  font-size: 14px;
+  font-family: 'Courier New', Courier, monospace;
+  color: whitesmoke;
+  margin: 5px;
+  font-weight: 600;
+}
+
+.main-feature h3 a,
+.sub-feature h3 a {
+  color: #fff;
+}
+
+.sub-feature {
+  flex-direction: column;
+}
+
+.tnw-card {
+  position: relative;
+  height: 100%;
+  width: 100%;
+}
+
+.overlay-main {
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: 0;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(148, 33, 33, 0.1);
+  background-image: linear-gradient(rgba(155, 148, 148, 0.2), rgba(0, 0, 0, 1));
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: 0;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(87, 86, 86, 0.2);
+  background-image: linear-gradient(to bottom, rgba(87, 86, 86, 0.2), rgba(0, 0, 0, 1));
+}
+
+.date {
+  display: none;
+}
+
+.tnw-img {
+  width: 100%;
+  height: 100%;
+}
+
+.caption {
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+  color: #fff;
+}
+
+.caption h3 a {
+  color: #fff;
+}
+
+.dropdown-content a:hover {
+  color: white;
+}
+
+.publication,
+span {
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+  color: lightgrey;
+  gap: 10px;
+  margin: 4px 0;
+}
+
+.highlighicon {
+  width: 16px;
+  height: 16px;
+}
+
+@media screen and (min-width: 822px) {
```

Comment:
[Optional]
- We have a repeating media query!! Thus `line 313` and `line 626`. Kindly fix this